src/test/scala/org/ergoplatform/appkit/ergotool/dex/CancelOrderSpec.scala

```diff
@@ -0,0 +1,97 @@
+package org.ergoplatform.appkit.ergotool.dex
+
+import org.ergoplatform.appkit.Parameters.MinFee
+import org.ergoplatform.appkit.impl.ErgoTreeContract
+import org.ergoplatform.appkit.{Address, ErgoContract, InputBox, MockInputBox, ObjectGenerators}
+import org.scalacheck.Gen
+import org.scalatest.{Matchers, PropSpec}
+import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
+import sigmastate.Values.SigmaPropConstant
+
+import scala.util.Failure
+
+class CancelOrderSpec extends PropSpec
+  with Matchers
+  with ScalaCheckDrivenPropertyChecks
+  with ObjectGenerators {
+
+  private def getOneInputBox(amountToSpent: Long): Seq[InputBox] =
+    Seq(MockInputBox(amountToSpent + unsignedLongGen.sample.get))
+
+  // TODO: extract to use in other commands
+  def sendToPk(address: Address): ErgoContract =
+    new ErgoTreeContract(SigmaPropConstant(address.getPublicKey))
```

Comment:
There is already similar method in BoxOperations.

Reply:
I believe you meant `ErgoContracts.sendToPk` - http://github.com/aslesarenko/ergo-appkit/blob/58aceae4f12d964523e7519db86b67a785b6ebd3/lib-api/src/main/java/org/ergoplatform/appkit/ErgoContracts.java#L14-L14
I saw it, but its dependency on `BlockchainContext` was a show-stopper.